python/requirements.txt

```diff
@@ -1,12 +1,15 @@
-numpy==1.13.0
-opencv-python==3.4.2.17
-pillow==7.1.0
-six==1.10.0
-scipy
+numpy>=1.13.0,<=1.19.0
+scikit-build<=0.11.1
+opencv-python>=3.4.2.17,<=4.0.0
+pillow>=6.2.2,<=7.1.2
```

Comment:
If I recall correctly, there were some security vulnerabilities with pillow library in 6.2. I would recommend using 7.* for minimum version too

Reply:
Correct, I recently updated python/requirements.txt to set
`pillow==7.1.0`
I did that because github is good about alerting when we have a deprecated or dangerous package/version. Github won't appreciate anything less than 7.1.0